activesupport/lib/active_support/core_ext/string/inquiry.rb

```diff
@@ -7,7 +7,7 @@ class String
   #   env = 'production'.inquiry
   #   env.production?  # => true
   #   env.development? # => false
-  def inquiry
-    ActiveSupport::StringInquirer.new(self)
+  def inquiry(*valid_values)
```

Comment:
Lets use a `restricted_to` optional keyword argument. `inquiry('pending', 'active')` doesn't read nice to me.